src/components/v5/common/EmptyContent/EmptyContent.tsx

```diff
@@ -39,12 +39,15 @@ const EmptyContent: FC<EmptyContentProps> = ({
         <p className="mt-2 text-sm text-gray-600">{descriptionText}</p>
         {onClick && (
           // @TODO: Change to Button component
-          <button type="button" className="mt-4" onClick={onClick}>
-            {withButtonIcon && (
-              <Icon name="share-network" appearance={{ size: 'normal' }} />
-            )}
+          <Button
+            mode="primaryOutline"
+            className="mt-4"
+            size="small"
+            iconName="share-network"
```

Comment:
what if there will be a case with a button without an icon?

Reply:
You can use:
- only label
- only icon
- both at the same time

If you don't use anything, the button will be empty
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/bf354831-c4c2-4d15-bd4b-545c3446785a)